streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -235,87 +226,99 @@
                     Type.LIST,
                     Importance.HIGH,
                     CommonClientConfigs.BOOTSTRAP_SERVERS_DOC)
-            .define(CLIENT_ID_CONFIG,
```

Comment:
The diff for the rest of the code seems to have gone wrong. Hard to tell what has changed and what the diff is. Might be ok.